wp-content/themes/core/assets/js/admin/editor/register-block-filter.js

```diff
@@ -0,0 +1,37 @@
+import { addFilter } from '@wordpress/hooks';
+
+// Specify block name and align settings for it to keep.
+const blockAlignments = {
```

Comment:
Similar here, It's be preferable if we had a way to set up these alignments within each block's directory rather than globally here.